src/ophys_etl/pipelines/brain_observatory/scripts/run_mesoscope_splitting.py

```diff
@@ -124,8 +130,9 @@ def split_image(input_tif, experiments, name):
     return outs, meta
 
 
-def split_timeseries(input_tif, experiments, **h5_opts):
-    mt = MesoscopeTiff(input_tif)
+def split_timeseries(input_tif: MesoscopeTiff,
+                     experiments: List[Dict],
+                     **h5_opts):
```

Comment:
do we really want/need to pass arbitrary h5 parameters through?
we tend to be minimally exlpicit, if possible.

Reply:
That's a good question. Since this isn't like a module, no other code should be importing this. So I'll just check if the ops get used anywhere in this script and make it minimal. Honestly, I think it'll mean just removing it altogether, I can't remember ever seeing a call use the options.

Reply:
We just discussed this in the meeting. Will address this in the next ticket.

Reply:
> We just discussed this in the meeting. Will address this in the next ticket.
I thought we discussed that it was a `1` for effort and we should finish it up here.

Reply:
Oh it sounded like there was a bigger production effort ticket coming up that this would get folded into. I'll just take care of it now.

Reply:
@djkapner @njmei Looking at the `input.json` for a couple random LIMS jobs in the `MESOSCOPE_FILE_SPLITTING_QUEUE` there are no references to compression. Is it safe to assume that it is never used, or should I go forward with converting `h5_opts` to a `dict` instead of `kwargs`?

Reply:
we're not using it, ever.
Can you just hard-code the h5 writing place to:
gzip and 4?

Reply:
Yep, I can!

Reply:
you still will need to pass on the chunks arg. please do so in an explicit way.

Reply:
Oh I think I misunderstand. If I'm forcing the file saving to always use those options, then why do I need to pass them around? Do you mean that I should do the hardcoding happen right here, not where the saving happens?